yaml/shalev/yaml_exercise3_$shalev.yaml

```diff
@@ -0,0 +1,36 @@
+step 1:
+  Ingredients:
+    - 4 cloves garlic
+    - 2 red chilli
+    - 3 cup water
+    - 225 gm pasta penne
+    - 1/2 tablespoon extra virgin olive oil
+
+  instruction: >
+  Heat oil in a pan over medium flame,
+  add minced garlic along with chopped red chilli and red chilli flakes,
+  Stir and saute for 2 minutes.
+  Take another pan, pour the water and bring it to a boil.
+  Then, add pasta to the boiling water along with some salt and 1/2 tbsp oil.
+  Cook the pasta for a few minutes. When it is cooked through,
+  drain the extra water and transfer the pasta to another bowl.
+
+step 2: &salt
+  Ingredients: salt
+  instruction: add a kurt of salt
+
+step 3: >
+  Now, heat the olive oil and add the peeled (or pureed) tomatoes, basil and parsley to the pan.
```

Comment:
keep object consistency of ingredients and instruction.
you can add ingredients: null